content/applications/hr/recruitment/add-new-applicants.rst

```diff
@@ -0,0 +1,175 @@
+:menuselection:`Recruitment` app dashboard. Then, click on a job position card. Next, click the
+:guilabel:`New` button in the top-left corner of that job position's :guilabel:`Applications` page.
+
+The :guilabel:`Subject / Application` field is populated with the pre-selected job position, by
+default. Certain fields on the applicant card may also be pre-populated, depending on how the job
+position is configured. Typically, the :guilabel:`Job` section, as well as the :guilabel:`Recruiter`
+field, are pre-populated.
+
+Enter the following information on the new applicant form. Note that not all fields listed below may
+be visible.
+
+.. note::
+   Depending on installed applications and configurations, some fields may **not** be displayed.
+
+.. _recruitment/applicant-details:
+
+Applicant section
+-----------------
+
+- :guilabel:`Subject/Application Name`: this is the **only** required field. Enter the title for the
+  card in this field. Typically, this is the applicant's name, and the job position being applied
+  to. For example: `John Smith - Experienced Developer`. This field is not visible in the Kanban
+  view of the :guilabel:`Applications` page, unless the :guilabel:`Applicant's Name` is left blank.
+  If there is no :guilabel:`Applicant's Name`, then the :guilabel:`Subject/Application Name` is what
+  is displayed on the applicant card.
+- :guilabel:`Applicant's Name`: enter the applicant's name.
+- :guilabel:`Email`: enter the applicant's email address.
+- :guilabel:`Phone`: enter the applicant's phone number.
+- :guilabel:`Mobile`: enter the applicant's mobile number.
+- :guilabel:`LinkedIn Profile`: enter the web address for the applicant's personal profile on
+  LinkedIn.
+- :guilabel:`Degree`: select the applicant's highest level of education from the drop-down menu.
+  Options are: :guilabel:`Graduate`, :guilabel:`Bachelor Degree`, :guilabel:`Master Degree`, or
+  :guilabel:`Doctoral Degree`. The :guilabel:`Graduate` option indicates the applicant graduated at
+  the highest level of school before a Bachelor's degree, such as a high school or secondary school
+  diploma, depending on the country.
+- :guilabel:`Interviewers`: using the drop-down menu, select the people to conduct the interviews.
+  The selected people **must** have either *recruiter* or *officer* rights configured for the
+  *Recruitment* application to appear in the drop-down list. Refer to the :doc:`Access rights
+  <../../general/users/access_rights>` documentation for more information.
+- :guilabel:`Recruiter`: select the user responsible for the entire recruitment process for the job
+  position.
+- :guilabel:`Evaluation`: represents a rating for the applicant: one star (:icon:`fa-star-o`
+  :icon:`fa-star-o` :icon:`fa-star`) is :guilabel:`Good`, two stars (:icon:`fa-star-o`
+  :icon:`fa-star` :icon:`fa-star`) is :guilabel:`Very Good`, and three stars (:icon:`fa-star`
```

Comment:
The display of the empty and filled in stars was backwards, updating to this so that filled in stars move from left to right
```suggestion
- :guilabel:`Evaluation`: represents a rating for the applicant: one star (:icon:`fa-star`
  :icon:`fa-star-o` :icon:`fa-star-o`) is :guilabel:`Good`, two stars (:icon:`fa-star`
  :icon:`fa-star` :icon:`fa-star-o`) is :guilabel:`Very Good`, and three stars (:icon:`fa-star`
```